src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -133,62 +108,32 @@ export const LongText: StoryObj<Args> = {
   },
 };
 
-export const LongButtonText: StoryObj<Args> = {
+export const LongTriggerText: StoryObj<Args> = {
   args: {
-    children: (
-      <Button
-        className={clsx(styles['trigger--spacing-top'])}
-        variant="primary"
-      >
-        Tooltip trigger with longer text to test placement
-      </Button>
-    ),
+    children: <div className="fpo p-1">Longer text to test placement</div>,
   },
 };
 
-export const DisabledButton: StoryObj<Args> = {
-  render: () => (
-    <div
-      className={clsx(
-        styles['trigger--spacing-top'],
-        styles['trigger--spacing-left'],
-      )}
-    >
-      <Tooltip align="top" childNotInteractive text={defaultArgs.text} visible>
-        <Button disabled variant="primary">
-          Tooltip trigger
-        </Button>
-      </Tooltip>
-    </div>
-  ),
-};
-
 export const TextChild: StoryObj<Args> = {
   render: () => (
-    <div
-      className={clsx(
-        styles['trigger--spacing-top'],
-        styles['trigger--spacing-left'],
-      )}
-    >
-      <Tooltip align="top" childNotInteractive text={defaultArgs.text} visible>
-        <span>Tooltip trigger</span>
-      </Tooltip>
-    </div>
+    <Tooltip align="top" childNotInteractive text={defaultArgs.text} visible>
+      <span>Tooltip trigger</span>
+    </Tooltip>
   ),
+  parameters: {
+    chromatic: {
+      diffThreshold,
+      diffIncludeAntiAliasing: false,
+    },
+  },
```

Comment:
Should inherit this from the default export, so we can probably remove these parameters

Reply:
Ah, well, `args` and `parameters` should be handled exactly the same.

Specifying either at the top level should cascade down to individual stories.

Reply:
@ahuth yep my bad, i meant that for some of those `render` examples, we write the test like 

```jsx

render: () => <SomeComponent />
```
in which case we ignore `args` even tho its available if we put `(args)` instead and spread those onto the right component

Reply:
OH, I see 👍 

Makes sense, carry on